var/spack/repos/builtin/packages/pdc/package.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Pdc(CMakePackage):
+    """Proactive Data Containers (PDC) software provides an object-centric
+    API and a runtime system with a set of data object management services.
+    These services allow placing data in the memory and storage hierarchy,
+    performing data movement asynchronously, and providing scalable
+    metadata operations to find data objects."""
+
+    homepage = "https://pdc.readthedocs.io/en/latest/"
+    url      = "https://github.com/hpc-io/pdc/archive/refs/tags/0.1.tar.gz"
+
+    maintainers = ['houjun', 'sbyna']
+
+    version('0.1', sha256='24787806a30cd1cda1fed17220a62e768bdba5de56877f2ea7126279ff2a4f69')
+
+    conflicts('%clang')
+    depends_on('libfabric@1.11.2')
```

Comment:
Do you really depend explicitly on one and only one minor version of this? It causes rebuilds for people who have other versions installed

Reply:
Can you please unconstrain this dependency?